setup/src/Magento/Setup/Test/Unit/Model/ConfigOptionsList/CacheTest.php

```diff
@@ -116,6 +122,60 @@ public function testCreateConfigWithRedisConfig()
         $this->assertEquals($expectedConfigData, $configData->getData());
     }
 
+<<<<<<< HEAD
```

Comment:
@orlangur I think this line is causing the Travis CL build to fail. Correct me if I'm wrong but this preserves the authorship so I don't think it can simply be removed.